src/quote/swap/interfaces.ts

```diff
@@ -25,6 +35,21 @@ export interface SwapQuoteRequest {
   sources?: Exchange[]
 }
 
+export interface SwapQuoteRequestV2 {
+  chainId: number
+  inputToken: string
+  outputToken: string
+  inputAmount: string
+  slippage: number
+  taker: string
+  // Optional
+  sources?: Exchange[]
+}
```

Comment:
New type to apply some changes e.g. certain properties now being required. 

Biggest change here 0x deprecated the `buyAmount`, so `outputAmount` can not be specified here any longer. For more infos on the decision: https://0x.org/docs/upgrading/upgrading_to_swap_v2#deprecated-parameter